### ISSUE TITLE
Address accessibility issues on brand guidelines website

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#234]
## Brief

Fix the WCAG fails that came out of the [accessibility audit](https://github.com/alphagov/govuk-brand-guidelines/issues/56) of the brand guidelines website.

We will be using an MVP approach in addressing these, using sensible minimal fixes.

Fix some of the other remaining accessibility issues, if we have time.

## Squad

@selfthinker, @calvin-lau-sig7, @querkmachine, @Izabela-16, @mia-allers-gds
